profile_library/signify/LCD008/model.json

```
@@ -0,0 +1,14 @@
+{
+  "calculation_strategy": "lut",
+  "created_at": "2024-12-05T07:26:13.311686",
+  "measure_description": "Measured with utils/measure script",
+  "measure_device": "Shelly PM Mini Gen3",
+  "measure_method": "script",
+  "measure_settings": {
+    "SAMPLE_COUNT": 2,
+    "SLEEP_TIME": 3,
+    "VERSION": "v1.16.2:docker"
+  },
+  "name": "LCD008",
```

This needs to be descriptive marketed model name, for example:

```suggestion
  "name": "Hue Xamento recessed light LED black",
```

It's shown in the name column in: https://library.powercalc.nl/

---

Name seems to be „Slim recessed light S“ white

---

Ok I have committed that.